packages/block-library/src/navigation/theme.scss

```diff
@@ -3,4 +3,45 @@
 	ul li {
 		list-style: none;
 	}
+
+	ul > li > ul {
```

Comment:
I'm 90% sure that this is necessary because menus can nest deeply, so you have to be specific in targetting. But are there any CSS classes you can use to target this instead? Like, will `.wp-block-navigation-item > ul` work?

The fewer items in selectors, the better!